### DISCUSSION TITLE
Installing on cpanel - are there instructions?

### DISCUSSION THREAD

**Discussion by CyberMinion** [the-djmaze/snappymail#671] [General]
Hello,

I recently discovered your project, and as a user of RainLoop, I'm interested! Unfortunately, it looks like on the install directions here:
"/the-djmaze/snappymail/wiki/Installation-instructions"
you do not yet have instructions for cpanel setups, which I am unfortunately stuck with for this deployment. Are they somewhere else, or have those not been written yet? If not, is there any way I can assist? I'm not too familiar with web applications, so I can't guarantee I'd be very helpful.

**Comment by cm-schl** [comment]
Hi,

don't know if you saw this, but maybe it can help you: https://github.com/the-djmaze/snappymail/issues/645

**Reply by CyberMinion** [reply]
Thanks for the reply! That looks to be more about how to replace the built-in webmail install in cpanel, with an already set up SnappyMail instance. That could be useful at some point, but I just need to install SnappyMail, first. I can upload the package, and handle the domain stuff. But then I need to handle permissions, and although I have limited terminal access, I cannot use the specified find functions to update permissions via chmod. So either I need documentation on what permissions level I should manually set each file to (one at a time), or some other way to do this, it seems.
With RainLoop, I was previously able to locate a list of file/folder permissions, and go through setting them manually one at a time. If something like that exists for this platform, or if there is a better method, that would be exceedingly helpful.
